classes/04class/exercises/c04-iac02/README.md

```diff
@@ -3,18 +3,22 @@
 In this exercise, you'll need to create a new VPC called `devopsacademy-iac`, but this time, every resource of this VPC will be created using Terraform. Similar to the [exercises done for class02](https://github.com/devopsacademyau/academy/tree/master/classes/02class/exercises).
 
 All your resources must be created in a single `vpc.tf` file.
+
 Your VPC will be composed by:
+
 - /16 CIDR of your choice in the 10.0.X.X range
-- 4 /24 Subnets, being 2 publics and  privates
+- 4 /24 Subnets, being 2 publics and privates
 - Internet GW attached to your VPC
-- 1 NAT GW (this will have costs. not mandatory to create)
-- Routing tables for the subnets be able to communicate witht he internet
+- 1 NAT GW (this will have costs). **Not mandatory to create**
```

Comment:
Thanks for the changes, they are welcome. However, could you please remove it from this PR and created a new PR only with the changes in the repo-content?

Reply:
Nice, I will.